cypress/integration/console/devices/create-manual.spec.js

```diff
@@ -390,12 +363,14 @@ describe.skip('End device manual create', () => {
         cy.findByLabelText('Network defaults').uncheck()
```

Comment:
```suggestion
        cy.findByLabelText('Network defaults').should('not.be.checked').and('have.attr', 'disabled')
```
The form will now actually automatically uncheck this when there is a required field within.